core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -171,7 +171,7 @@ public CallOptions withAffinity(Attributes affinity) {
 
   /**
    * Enables 'wait for ready' feature for the call.
-   * <a href="https://github.com/grpc/grpc/blob/master/doc/fail_fast.md">'Fail fast'</a>
+   * <a href="https://github.com/grpc/grpc/blob/master/doc/wait-for-ready.md">'wait for ready'</a>
```

Comment:
You can leave `Fail fast` without link and put the link at `wait for ready` in the next line instead:)